addon/components/weekly-events.hbs

```diff
@@ -1,9 +1,23 @@
-<div class="weekly-events">
-  <div class="year">
+<div class="weekly-events" data-test-weekly-events>
+  <div class="year" data-test-top-nav>
     <h2>
-      <FaIcon @icon="backward" {{on "click" this.decrementYear}}/>
-      {{@year}}
-      <FaIcon @icon="forward" {{on "click" this.incrementYear}}/>
+      <FaIcon
+        @icon="backward"
+        @title={{t "general.goToPreviousYear"}}
+        role="button"
```

Comment:
Passing `@click` to the `<FaIcon` will add the necessary a11y markup instead of needing to handle it here.

Reply:
sorry to disappoint, but that's incorrect. if no `role` attribute is passed then `role` will default to `img`.
![Selection_232](https://user-images.githubusercontent.com/1410427/94641209-7fa78600-0295-11eb-84c1-321383e75132.png)

Reply:
Oh no! Well that's not good. Ok.